DataHub/Operation/03_ManageHTMLTemplates.md

```diff
@@ -0,0 +1,296 @@
+[!!User interface HTML templates](../UserInterface/02j_HTMLTemplates.md)
+
+# Manage the HTML templates
+
+Define HTML templates that are used for the ETL mapping, especially for the receipt export.
```

Comment:
html templates are mostly used for emailing like confirmation or shipped message

and could be used to compose rich product description when uploading to ebay for example